lib/login-providers/browser.js

```diff
@@ -76,7 +76,8 @@ class BrowserLoginProvider extends LoginProviderBase {
             reject(body.error_description);
             return;
           }
-          resolve(super.storeJWT(body.id_token));
+          // Must use this.storeJWT instead of super.storeJWT to support bug in Node 4.
+          resolve(this.storeJWT(body.id_token));
```

Comment:
I know that arrow functions don't specify their own `this` or `super`, so these keywords refer to the `this` or `super` from the next outer closure (if any). I had no idea this was broken in V8 for a while there. Fascinating. :D